README.md

```diff
@@ -1,158 +1,236 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+description: "This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure."
+urlFragment: key-vault-node-quickstart
+---
 
 # Quickstart: Set and retrieve a secret from Azure Key Vault using a Node Web App 
 
-This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure. The quickstart uses Node.js and Managed service identities (MSIs)
+This Quickstart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure. The Quickstart uses Node.js and Azure Managed Identities
```

Comment:
Add a period at the end, please link to the Managed Identities docs page. https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/